### ISSUE TITLE
Boot not working with Jetpack 4.5.1

### ISSUE THREAD

**Issue by jodur** [jetsonhacks/bootFromUSB#1]
After watching the latest Jetson AI labes, where the new Boot from Usb was announced i moved over to the github to give it a try. Unfortunally it doesn't work for me. After powerup i get the nvidea logo with  a SD-card image with a red dash, meaning no SD card inserted, afterward nothing happening.

**Comment by jetsonhacks** [comment]
Which Jetson are you using, and what does your extlinux.conf file look like?

**Comment by jodur** [comment]
When using the script  `./partUUID.sh` to get the PARTID it returned an empty string. Also the device was listed as `/dev/sda` instead of `/dev/sda1`. I ignored this first and used the disk UUID instead. (RTFM was well done be me, sorry). I started over again and installed `gparted` to create the partion. This time the device was listed as `/dev/sda1` and the script now also return a partUUID. When i used that in the `extlinux.conf` file, the boot from disk worked fine. I started with a fresh SSD drive, maybe that could ne the reason for the weird behaviour? Thank for youre manual and sorry for bothering you.
